### DISCUSSION TITLE
Would this work on an Optiplex 990?

### DISCUSSION THREAD

**Discussion by nielsfeels** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#63] [Q&A] [answered]
Dear all, thanks a lot for this guide, it worked perfectly on my 3020. I have got a 990, with an i7 2700, would this guide work with it as well? Or could you point me to the direction what I would have to change to make it work?

best regards and thanks again
-niels

**Comment by JohnAker33** [comment, marked as answer]
The 990 is a Sandy Bridge system with a different motherboard chipset as well. The 3020 is Haswell based so it will not work with the included EFI. Have a look at the Github Dortania guide with emphasis on the Sandy Bridge section.

**Reply by Xtendera** [reply]
I cannot find a "guide" like this one on the CPU model, probably because it does not have iGPU support starting with Mojave. You could try to get High Sierra working by making your own EFI using the [Opencore guide](https://dortania.github.io/OpenCore-Install-Guide/) as suggested by JohnAker33.

**Reply by JohnAker33** [reply]
Will require a macOS supported dGPU for that hardware to work as a hackintosh.

**Comment by nielsfeels** [comment]
Thank you!
